docker/compose.yml

```diff
@@ -152,7 +152,7 @@ services:
       - zookeeper
 
   cloudbeaver:
-    image: pan3793/cloudbeaver:20240223
+    image: dbeaver/cloudbeaver:24.1
```

Comment:
switch to official dbeaver image as i already pushed the custom changes into upstream